specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json

```diff
@@ -50,7 +50,7 @@
         }
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {

```

Comment:
Thankyou. Removing SubsId and ResourceGroupName from parameter from subsequent commit.